server/index.js

```diff
@@ -20,16 +21,17 @@ const userRouter = require('./routes/userRoutes');
 const app = express();
 app.use(cors());
 app.use(helmet());
-app.use(express.json());
+// app.use(express.json());
+// app.use(express.json({ limit: '50mb' }));
+// app.use(express.urlencoded({ limit: '50mb', extended: true }));
+app.use(bodyParser.json({limit: '50mb'}));
+app.use(bodyParser.urlencoded({limit: '50mb', extended: true}));
```

Comment:
tried to upload a video and it said that the payload was too large, unsure if this needs to be increased